BE/src/main/java/com/sidedish/repository/CategoryRepository.java

```diff
@@ -10,5 +10,5 @@
 public interface CategoryRepository extends CrudRepository<Category, Long> {
 
     @Query("SELECT id FROM category WHERE type=:type")
-    Long findCategoeyId(@Param("type") CategoryType type);
+    Long findCategoryType(@Param("type") CategoryType type);
```

Comment:
categoryType으로 조회 시 조금 더 보편적인 네이밍은 아래처럼 사용합니다. ㅎㅎ

```suggestion
    Long findByCategoryType(@Param("type") CategoryType type);
```